var/spack/repos/builtin/packages/butterflypack/package.py

```diff
@@ -22,10 +22,11 @@ class Butterflypack(CMakePackage):
 
     homepage = "https://github.com/liuyangzhuan/ButterflyPACK"
     git      = "https://github.com/liuyangzhuan/ButterflyPACK.git"
-    url      = "https://github.com/liuyangzhuan/ButterflyPACK/archive/v2.1.0.tar.gz"
+    url      = "https://github.com/liuyangzhuan/ButterflyPACK/archive/v2.1.1.tar.gz"
```

Comment:
It's not necessary to do this in the future: spack just uses this as a template for the download URL.

Reply:
Does that mean that we don't need to specify "url" in future?